ray-operator/controllers/ray/common/pod.go

```diff
@@ -170,7 +170,7 @@ func BuildAutoscalerContainer() v1.Container {
 		// TODO: choose right version based on instance.spec.Version
 		// The currently used image reflects changes up to https://github.com/ray-project/ray/pull/24718
 		Image:           "rayproject/ray:448f52",
-		ImagePullPolicy: v1.PullAlways,
+		ImagePullPolicy: v1.PullIfNotPresent,
```

Comment:
theoretically a breaking change for users who use `rayproject/ray:latest` but we aren't GA and that's a bad practice so it seems fine.

Should we have some form of documentation somewhere to not use the `latest` and `nightly` tags for long running clusters?

Reply:
The initial version was merged like yesterday, so we're fine from a backwards compatibility standpoint.
There are no users using `rayproject/ray:latest`.

The docs are not written yet, but they will explain that the image and pull policy are for testing/dev and not to be modified for production use-cases.

Reply:
> Should we have some form of documentation somewhere to not use the latest and nightly tags for long running clusters?

Latest and nightly tags should be used **never**. We can include a reminder in the docs that it's bad practice.

Reply:
Hmm, I'll just change it back to PullAlways to be conservative -- for a pinned image, it's immaterial.